### ISSUE TITLE
5 Cookiemonster additions

### ISSUE THREAD

**Issue by practik** [ryanbr/fanboy-adblock#520]
- accessmedicine.mhmedical.com##.widget-instance-MGH_CookieAcceptance.widget-GdprCookieBanner
- bandcamp.com###menubar-verify-banner
- bl.uk###cookies
- nationalgalleries.org##.ngs-cookie-bar--show.ngs-cookie-bar
- staatsbibliothek-berlin.de###tracking-info

The Bandcamp one is hard to reproduce, since it seems to show up only occasionally – maybe once a day? I'm not on the site a lot, so it's hard to tell. I first saw the banner the last time I was there, a couple of weeks ago, and then got it again today on my first visit to the site:

![bandcamp-gdprbanner](https://user-images.githubusercontent.com/12074235/43860443-f305b200-9b10-11e8-9030-d58b8e25a6e7.png)

**Comment by krystian3w** [comment]
`bandcamp.com##.corpbanner.tos-update` - this is used by "`I DON'T CARE ABOUT COOKIES`"

**Comment by ryanbr** [comment]
https://github.com/easylist/easylist/commit/6dfc3d7f48561d18c18abebfa102d138bf9ba453
https://github.com/easylist/easylist/commit/0466819373a5beaa6051541b33a00b66cbc93dc1
https://github.com/easylist/easylist/commit/5fc8cb02986956970848907f621758a86968152f
https://github.com/easylist/easylist/commit/18f5b4d52be99334fdb01ac72fa4fd5c4329f5d2
https://github.com/easylist/easylist/commit/dfb558605bd8a7f9c1f92f7acd0137254583783d

**Comment by practik** [comment]
Just noticed a mistake in easylist/easylist@0466819, actually:  The class should be `corpbanner`, not `corp-banners`.

**Comment by ryanbr** [comment]
I see it being hit, and no cookie banner. Keep in mind div elements will have many inner elements with different name.

**Comment by practik** [comment]
> I see it being hit

Sorry, I don't know what this means.

> This is traffic on the part of the webmaster

Meaning that Bandcamp used to have a `corp-banners` class but changed its name sometime in the last month? That's possible. But my screenshot from a month ago shows `corpbanner`, not `corp-banners`.

> no cookie banner

I'm not seeing the banner today either, but I did see it three days ago when I posted my last comment. At that time I searched the HTML of the entire page – I'm aware that divs with different names can be nested – and didn't find `corp-banners` anywhere, just `corpbanner` inside `menubar-verify-banner` inside `bannercontainer` like before. So I assumed you mistyped the class name. Sorry about that.
